addons/survey/models/survey_survey.py

```diff
@@ -1210,54 +1241,46 @@ def _create_preview_image(self, for_results):
                 break
             else:
                 title_lines.append(word)
-        d.text((m, m), "\n".join(title_lines), font=title_font, fill="darkgrey")
+        d.text((m, m + icon.height), "\n".join(title_lines), font=title_font, fill=title_color)
 
-        # User picture and create date
-        x, y = m, title_line_h * len(title_lines) + font_size
+        # User picture, user name and create date
+        x, y = m, h - m - font_size;
         if self.user_id.image_128:
             avatar = b64decode(self.user_id.image_128)
         else:
             avatar = self.env['avatar.mixin']._avatar_get_placeholder()
-        user_img = Image.open(BytesIO(avatar)).resize((title_line_h, title_line_h))
+        user_img = Image.open(BytesIO(avatar)).resize((avatar_size, avatar_size))
         back_color = Image.new(user_img.mode, user_img.size, "white")
         mask = Image.new("L", user_img.size, 0)
         d2 = ImageDraw.Draw(mask)
-        d2.ellipse((0, 0, *user_img.size), fill=255)
+        d2.rounded_rectangle((0, 0, *user_img.size), fill="white", outline=None, width=4, radius=8)
         rounded_user = Image.composite(user_img, back_color, mask)
         img.paste(rounded_user, (x, y))
```

Comment:
Concerning the issue of the borders not being transparent, the following code seems to fix it:
```py        
mask = Image.new("RGBA", user_img.size, 0)
d2 = ImageDraw.Draw(mask)
d2.rounded_rectangle((0, 0, *user_img.size), fill="white", outline=None, width=4, radius=8)
img.paste(user_img, (x, y), mask)
```